wagtail/admin/tests/test_rich_text.py

```diff
@@ -491,3 +493,87 @@ def test_versioned_static_media(self):
         media_html = str(feature.media)
         self.assertRegex(media_html, r"feature.js\?v=(\w+)")
         self.assertRegex(media_html, r"feature.css\?v=(\w+)")
+
+
+class TestEntityFeatureChooserUrls(TestCase):
+    def test_chooser_urls_exist(self):
+        features = FeatureRegistry()
+        image = features.get_editor_plugin("draftail", "image")
+        embed = features.get_editor_plugin("draftail", "embed")
+        document = features.get_editor_plugin("draftail", "document-link")
+        link = features.get_editor_plugin("draftail", "link")
+
+        # imageChooser
+        self.assertIsNotNone(image.data.get("chooserUrls"))
+        self.assertEqual(
+            image.data["chooserUrls"]["imageChooser"],
+            reverse_lazy("wagtailimages_chooser:choose"),
+        )
+        # linkChoosers
+        self.assertIsNotNone(link.data.get("chooserUrls"))
+        self.assertEqual(
+            link.data["chooserUrls"]["pageChooser"],
+            reverse_lazy("wagtailadmin_choose_page"),
+        )
+        self.assertEqual(
+            link.data["chooserUrls"]["externalLinkChooser"],
+            reverse_lazy("wagtailadmin_choose_page_external_link"),
+        )
+        self.assertEqual(
+            link.data["chooserUrls"]["emailLinkChooser"],
+            reverse_lazy("wagtailadmin_choose_page_email_link"),
+        )
+        self.assertEqual(
+            link.data["chooserUrls"]["phoneLinkChooser"],
+            reverse_lazy("wagtailadmin_choose_page_phone_link"),
+        )
+        self.assertEqual(
+            link.data["chooserUrls"]["anchorLinkChooser"],
+            reverse_lazy("wagtailadmin_choose_page_anchor_link"),
+        )
+        # documentChooser
+        self.assertIsNotNone(document.data.get("chooserUrls"))
+        self.assertEqual(
+            document.data["chooserUrls"]["documentChooser"],
+            reverse_lazy("wagtaildocs_chooser:choose"),
+        )
+        # embedsChooser
+        self.assertIsNotNone(embed.data.get("chooserUrls"))
+        self.assertEqual(
+            embed.data["chooserUrls"]["embedsChooser"],
+            reverse_lazy("wagtailembeds:chooser"),
+        )
```

Comment:
We should not be testing documents/embeds/images in the core tests, we have to assume these are isolated modules and may not always be included.

Please find a suitable place for these in the specific module tests.

Eg. `wagtail/embeds/tests/test_rich_text.py`, `wagtail/images/tests/test_rich_text.py` & `wagtail/documents/tests/test_rich_text.py`.